src/client/components/StudyGroupLogo/StudyGroupLogo.js

```diff
@@ -5,16 +5,16 @@ import './StudyGroupLogo.styles.css';
 
 export const StudyGroupLogo = (props) => {
   return (
-    <i>
-      <img src={props.path} width="50" alt="logo" />
-    </i>
+    <span className="rounded-circle">
+      <img src={props.path} width="50" alt="logo" className="logos" />
```

Comment:
I hate to nitpick, but it is `logo` - it is just one!

Reply:
removed